### PR TITLE
Add export command to dem

### PR DESCRIPTION
**Pull request by magyta93**

### CODE REVIEW

**Review by janosmurai — CHANGES_REQUESTED** [review]
The following options don't work (Assuming the Tutorial Dev Env is available):
`dem export Tutorial ~/test.json`
`dem export Tutorial`

File: dem/cli/main.py [hunk 1]
```diff
@@ -62,6 +62,14 @@ def create(dev_env_name: str = typer.Argument(...,
     """
     create_cmd.execute(dev_env_name)
 
+@typer_cli.command()
+def export(dev_env_name: str = typer.Argument(...,help="Name of the Development Environment to export."),
+           path_to_export: str = typer.Argument(...,help="Path where to extract the Dev Env.")) -> None:
```
[the remark this answers is not in the document]
Comment (magyta93) [reply dem/cli/main.py]: fixed

File: dem/cli/command/export_cmd.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from dem.cli.console import stderr
+import json, os
+
+def check_is_file(param):
+     return(os.path.isfile(param))
+     
+def check_is_directory(param):
+     return(os.path.isdir(param))     
+
+def create_exported_dev_env_json(dev_env_name,dev_env_json,given_path):
```
[the remark this answers is not in the document]
Comment (magyta93) [reply dem/cli/command/export_cmd.py]: fixed

File: dem/cli/command/export_cmd.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"""export CLI command implementation."""
+# dem/cli/command/export_cmd.py
+
+
+import re
+from dem.core.dev_env_setup import DevEnvLocalSetup
+from dem.cli.console import stderr
+import json, os
+
+def check_is_file(param):
```
[the remark this answers is not in the document]
Comment (magyta93) [reply dem/cli/command/export_cmd.py]: fixed

File: tests/cli/test_export_cmd.py
[the remark this answers is not in the document]
Comment (magyta93) [reply tests/cli/test_export_cmd.py:1]: fixed

**Review by janosmurai — APPROVED** [review]

**Pull request merged**